regsync/Dockerfile

```diff
@@ -13,4 +13,4 @@ RUN pip install -e .
 RUN chown -R 1000:1000 /app
 USER 1000
 
-ENTRYPOINT ["python"]
+ENTRYPOINT ["bridge"]
```

Comment:
Add a default command of `run` so that a plain docker run does the 'right thing'?